roles/ci_gen_kustomize_values/molecule/default/prepare.yml

```diff
@@ -20,7 +20,7 @@
   vars:
     cifmw_ci_gen_kustomize_values_src_dir: >-
       {{
-        (ansible_user_dir,
+        (lookup('env', 'HOME', '/home/zuul'),
```

Comment:
question, I assume that `/home/zuul/` is the default value, don't we need a `default` keyword? The examples in the docs all seem to use it https://docs.ansible.com/ansible/latest/collections/ansible/builtin/env_lookup.html

Reply:
It seems to work without the keyword - at least molecule isn't failing with that, and seems to provide the correct value